hasher-matcher-actioner/terraform/actions/main.tf

```diff
@@ -265,6 +272,14 @@ data "aws_iam_policy_document" "action_performer" {
     actions   = ["secretsmanager:GetSecretValue"]
     resources = [var.te_api_token_secret.arn]
   }
+
+  statement {
+    effect = "Allow"
+    actions = [
+      "dynamodb:Get*",
```

Comment:
The permission I used was scan, not get. Got that from the fetcher I though?

Reply:
Hmm. Get* works for me. Any reason to change?

Reply:
consistency?